keras2onnx/_builtin.py

```diff
@@ -524,6 +527,40 @@ def convert_tf_gather_nd(scope, operator, container):
                               name=operator.full_name)
 
 
+def _convert_tf_compare_equal(scope, operator, container, tf_op_string, onnx_op_string):
+    if operator.target_opset < 7:
+        raise ValueError(tf_op_string + " op is not supported for opset < 7")
+    oopb = OnnxOperatorBuilder(container, scope)
+    if operator.target_opset >= 9:
+        compare_node = oopb.add_node(onnx_op_string,
+                                     operator.input_full_names,
+                                     operator.full_name + '_' + onnx_op_string.lower())
+        oopb.add_node_with_output('Not',
```

Comment:
I assume this "Not" is to address the not >= or the <= case, but why is it added to the graph in every case? Shouldn't it be dependent on the result of the comparison?

Reply:
Addressed offline. GreaterEqual = Less + Not and vice versa.